src/validation.h

```diff
@@ -355,8 +355,7 @@ bool CheckSequenceLocks(const CTransaction &tx, int flags, LockPoints* lp = null
 class CScriptCheck
 {
 private:
-    CScript scriptPubKey;
-    CAmount amount;
+    CTxOut out;
```

Comment:
nit: I know it's tedious, but can you name this according to the new style guide. In this case `m_tx_out`

Reply:
Where could I see the new style guide? And what does the "m" mean? Actually I think "prevout" is the most accurate name but the name is already used.

Reply:
https://github.com/bitcoin/bitcoin/blob/master/doc/developer-notes.md

> Class member variables have a m_ prefix.

Reply:
actually the name `out` is taken from https://github.com/bitcoin/bitcoin/blob/v0.15.0.1/src/coins.h#L33 . So for consistency we have to keep it.

Reply:
I don't understand. That's for `Coin` class member. Why does the `CScriptCheck` member need to be named the same simply because it's of the same type?